cypress/e2e/0-JoinBet/registration/registration.cy.js

```diff
@@ -37,4 +38,80 @@ describe('Registration. Localization. Casino. Header.',() => {
                 .click();
         });
     });
+});
+
+describe('Registration. Casino. Header.', () => {
+    beforeEach(() => {
+        cy.authenticateUrl('https://st2-gdjsmns.hk/ru/casino/lobby');
+        bonusModalClose();
+    });
+    /*
+    C4928 - https://jbtest.testrail.io/index.php?/cases/view/4928
+     */
+    it('Проверить открытие/закрытие дропдауна "Валюта"', () => {
+        cy.get(header.regButton)
+            .click();
+        cy.get(header.window1.currencyItem)
+            .click();
+        cy.get(header.window1.currencyList)
+            .should('be.visible')
+            .get(header.window1.banner)
+            .click()
+            .get(header.window1.currencyList)
+            .should('not.be.visible');
+        cy.get(header.closeRegButton)
+            .click();
+    });
+    /*
+    C1501 - https://jbtest.testrail.io/index.php?/cases/view/1501
+     */
+    it('Нажать на кнопку "Регистрация по телефону" после заполнения полей',() => {
+        cy.get(header.regButton)
+            .click();
```

Comment:
@ElKalinin Дружище, чтобы как-то сократить кол-во строк - предлагаю ГДЕ ЭТО ВОЗМОЖНО написать в 1 строчку 
```suggestion
cy.get(header.regButton).click();
```

Reply:
done